src/plugins/intel_cpu/src/config.h

```diff
@@ -62,20 +62,22 @@ struct Config {
     bool valueCachePrecisionSetExplicitly = false;
     bool keyCacheGroupSizeSetExplicitly = false;
     bool valueCacheGroupSizeSetExplicitly = false;
-#if defined(OV_CPU_WITH_ACL)
+#if defined(OV_CPU_WITH_ACL) || defined(OV_CPU_WITH_SHL)
     bool aclFastMath = false;
```

Comment:
I believe that `aclFastMath` is used only by ACL and should be defined only when `defined(OV_CPU_WITH_ACL)` - without any `defined(OV_CPU_WITH_SHL)`. Could you please define `aclFastMath` only when `defined(OV_CPU_WITH_ACL)` as it was previously?

Reply:
Done